app/views/config_groups/index.html.erb

```diff
@@ -8,17 +8,17 @@
   <thead>
     <tr>
       <th class="col-md-6"><%= sort :name %></th>
-      <th><%= sort :config_group_classes_count, :as => _('Puppet classes'), :default => "DESC" %></th>
-      <th><%= sort :hosts_count, :as => _('Hosts'), :default => "DESC" %></th>
-      <th><%= sort :hostgroups_count, :as => _('Host groups'), :default => "DESC" %></th>
+      <th><%= _('Puppet classes') %></th>
+      <th><%= _('Hosts') %></th>
+      <th><%= _('Host groups') %></th>
       <th><%= _('Actions') %></th>
```

Comment:
We lose sorting on these columns (and for other resources as well) but I agree this is the correct approach.